docs/visual-testing/sauce-visual.md

```diff
@@ -3,3 +3,45 @@ id: sauce-visual
 title: Sauce Labs Visual Testing
 sidebar_label: Getting Started
 ---
+
+## Introduction
+
+Sauce Visual helps you compare uploaded images (snapshots) against reference images (baselines). It offers APIs to upload snapshots and compare those against baselines. The result of the comparison process includes the details of eventually detected differences. All snapshots in Sauce Visual need to be assigned to visual builds.
+
+## Workflow
+
+To benefit from Sauce Visual, you typically add it to your existing automated tests using provided libraries. We provide bindings for:
+- JavaScript/TypeScript
+- Java
+You can use those standalone or with your Selenium/Appium-based tests. Support for other frameworks like Cypress, StoryBooks, and Playwright is on the Roadmap.
+
+The best way to integrate Sauce Visual into your existing tests (or write new ones) is to follow the examples listed [in the repository](../README.md).
+
+After you have executed your tests, you will find your Visual test results on the Builds History Page:
+
+![Diff History Page}](./resources/BuildHistoryPage.png)
+
+Selecting one of the builds allows you to get the Diff Review Page, where you can Approve/Reject detected diffs.
+
+
+![Diff Review Page}](./resources/DiffReviewPage.png)
```

Comment:
@KunalJain, the image is missing and should be placed in the static/img folder inside sauce-docs. Can you please fix it?